src/data-scraper/index.ts

```diff
@@ -52,25 +59,23 @@ export async function validateConnectivity(): Promise<void> {
 
 export async function scrapeData(): Promise<void> {
   const url = getSysdigUrl();
-  const header = getSysdigAuthHeader();
+  const header = getSysdigAuthToken();
+  const clusterName = getSysdigClusterName();
 
-  // limit: min 1, max 500, default 250
-  const limit: number = 10;
   const reqOptions: NeedleOptions = {
     agent: httpsAgent,
     headers: {
       authorization: header,
     },
   };
 
-  let cursor: string = '';
   while (true) {
     try {
-      logger.info({ cursor }, 'attempting to get runtime images');
+      logger.info('attempting to get runtime images');
 
       const { response, attempt } = await retryRequest(
         'get',
-        `${url}?limit=${limit}&cursor=${cursor}`,
+        `${url}?clusterName=${clusterName}`,
```

Comment:
is this API not paginated then?

Reply:
I'm getting a 500 when using pagination as before - so I'd say yes.

Reply:
interesting, maybe ask on the channel for that, the payloads can be very large

Reply:
if we are not paginating we can remove the while(){} in L70